pages/common/git-force-clone.md

```diff
@@ -0,0 +1,13 @@
+# git force-clone
+
+> Provides the basic functionality of `git clone`, but if the destination git repository already exists it will force-reset it to resemble a clone of the remote.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-force-clone>.
+
+- Clone a Git repository into a new folder:
```

Comment:
The `-b` argument is not required, could you add a example without it?

Reply:
Of course! Will do!

Reply:
I have added a new example at the top without the optional `-b` argument. I also have specified in the other examples that `-b` checkouts an specific branch. 
I have considered to replace the samples in the last command example for placeholders, but I think it is better in this way (because of the existing folder behaviour).

What do you think about this changes? @navarroaxel